lib/katello/version.rb

```diff
@@ -1,3 +1,3 @@
 module Katello
-  VERSION = "3.10.0".freeze
+  VERSION = "3.10.0.master".freeze
```

Comment:
We can see what the tests say, but I think this may have to be a dash similar to how hammer-cli-foreman does it:

https://github.com/theforeman/hammer-cli-foreman/blob/master/lib/hammer_cli_foreman/version.rb#L3

Reply:
According to the [rubygem-katello.spec](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/katello/rubygem-katello/rubygem-katello.spec#L19) it looks like it just wants `version` and `prerelease` smashed together.